pkg/storage/export/export/export.go

```diff
@@ -599,6 +608,53 @@ func (ctrl *VMExportController) createCertSecretManifest(vmExport *exportv1.Virt
 	}, nil
 }
 
+// handleVMExportSecret checks if a secret has been specified for the current export object and, if not, creates one specific to it
+func (ctrl *VMExportController) handleVMExportSecret(vmExport *exportv1.VirtualMachineExport) error {
+	// If a tokenSecretRef has been specified, we assume that the corresponding
+	// secret has already been created and managed appropiately by the user
+	if isTokenSpecified := vmExport.Spec.TokenSecretRef != ""; isTokenSpecified {
+		return nil
+	}
+
+	// The secret name is constructed so it can be specific to the current vmExport object
+	tokenSecretName := getDefaultTokenSecretName(vmExport)
+
+	token := rand.String(secretTokenLenght)
```

Comment:
I think we have to use a more cryptographically secure rng for the token.  Looks like this package user "math/rand" which "implements pseudo-random number generators unsuitable for security-sensitive work".

Reply:
hmmm I'm honestly surprised by that since I thought  `k8s.io/apimachinery/pkg/util/rand` was our standard for this kind of thing. I see that `/vendor/k8s.io/client-go/util/cert` uses `crypto/rand`, so I'll find a way to use that one instead (also in virtctl vmexport).

Reply:
Yeah, math/rand is fine for naming things but not for secrets